src/jemalloc.c

```diff
@@ -1026,6 +1029,7 @@ obtain_malloc_conf(unsigned which_source, char buf[PATH_MAX + 1]) {
 			 * Do nothing; opts is already initialized to the value
 			 * of the MALLOC_CONF environment variable.
 			 */
+			malloc_conf_env_var = ret;
```

Comment:
nit: update the comments above as well (the do nothing part isn't true anymore).